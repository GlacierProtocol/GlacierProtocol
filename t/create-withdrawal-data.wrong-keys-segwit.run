#!/bin/bash
set -e

# Test validation of extra keys. 3rd key does not belong to this address.


../../glacierscript.py --testnet=$1 create-withdrawal-data << INPUT
y
y
y
y
y
y
2Mv6fmwU6Nj6tiSNHzxSDz8vFiEJkYQ46Fm
5221027cac2fe4bb16fa9cdd57de980f546d7cddd8704f11dfc37803ce8831955caf3821031344a608e2a2e7f0a4fd3f60e004f633ef4361482b5bbf4e420a07035ca3554121023d6acb654efdd5b6354dd6bfa2e1fab9feab6468d970fdac9b373dfadb11d5952103a6c60cec3e79f91bec8453f534501f05ad2b0b6e6e1faae05e7a2391d65d998054ae
2NBNqB9wqYYxgcCSeqNrR9RmHc4NsLm7jWe
1
02000000000101c1ce3ff7c69a7e83b44b97fa9600d2be505fc50b59c04dbfaad213e822cf8dbc00000000171600148054baa5f0b5c8a013752499a0f0258d8c373893fdffffff02dbfaea000000000017a914eebaac00268c662200af795450b94e7f9e5274768780841e000000000017a9141f4a28e4da99092418131207521012b19cd45f3e870247304402200d50548ef9b682c2e5e67f805b5b04470df3a10d526007fe2fb7fae692cec21702204875af62e0ba936d80182352699e9196351284f2cbe1162c4ed3499851a3fd0d012102917583a635d983fb8ad6336964254ae81a952ab7fbe3bdbacaba02ef8e1a8ed7a0351600
4
cUY5DRXSmpqT7WpBAkjFz6PPxHoPJ2Cw2VNCa7muihpRjAUJVycs
cQTQEE1ooQHpVi5C4aTcbYPSUMzY2gF1DVdafUsEwnYYaT97UUMy
cPSsBu9SyNVAS2Evy3m4ELFx7KGnudH3N77Es83nafa2xVWJGRSe
cTQHU4ZzwLthJhDU6oxSzmWsTmwhDvFphJCfKFDZxxy6F8j37jRq
10
y

y
INPUT
