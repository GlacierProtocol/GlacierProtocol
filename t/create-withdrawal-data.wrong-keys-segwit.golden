Are you running this on a computer WITHOUT a network connection of any kind? (y/n)?Have the wireless cards in this computer been physically removed? (y/n)?Are you running on battery power? (y/n)?Are you running on an operating system booted from a USB drive? (y/n)?Is your screen hidden from view of windows, cameras, and other people? (y/n)?Are smartphones and all other nearby devices turned off and in a Faraday bag? (y/n)?
You will need to enter several pieces of information to create a withdrawal transaction.


*** PLEASE BE SURE TO ENTER THE CORRECT DESTINATION ADDRESS ***


Source cold storage address: 
Redemption script for source cold storage address: 
Destination address: 
How many unspent transactions will you be using for this withdrawal? 
Please paste raw transaction #1 (hexadecimal format) with unspent outputs at the source address
OR
input a filename located in the current directory which contains the raw transaction data
(If the transaction data is over ~4000 characters long, you _must_ use a file.):

Transaction data found for source address.
TOTAL unspent amount for this raw transaction: 0.02000000 BTC

How many private keys will you be signing this transaction with? 
#: Key #1: Key #2: Key #3: ERROR: that key does not belong to this source address, exiting...
